layouts/shortcodes/partners.html

```diff
@@ -0,0 +1,34 @@
+<div class="partner-images">
+    <a href="https://www.quansight.com">
+        <img src="/images/logos/quansight.svg" width="200" alt="Logo of Quansight" />
```

Comment:
This is the old logo. I don't have the new Quansight one at hand, but for now can you use this Quansight Labs one instead?

https://github.com/Quansight-Labs/quansight-labs-site/blob/main/files/images/QuansightLabs_logo_V2.png